js-functions/js-functions-2_arrow-functions/js/index.js

```diff
@@ -19,14 +19,36 @@ function getCurrentHour() {
   }
 }
 
+const getCurrentHourArrow = () => {
+  const now = new Date();
+  const currentHour = now.getHours();
+  if (currentHour === 0) {
+    return "12am";
+  } else if (currentHour === 12) {
+    return "12pm";
+  } else if (currentHour <= 11) {
+    return currentHour + "am";
+  } else {
+    return currentHour - 12 + "pm";
+  }
+};
+
 function getVectorLength(x, y, z) {
   return (x ** 2 + y ** 2 + z ** 2) ** 0.5;
 }
 
+const getVectorLengthArrow = (x, y, z) => {
+  return (x ** 2 + y ** 2 + z ** 2) ** 0.5;
+};
+
 function cleanInput(string) {
   return string.toLowerCase().trim();
 }
 
+const cleanInputArrow = (string) => {
```

Comment:
if there is only one parameter we can skip the parenthesis of the parameter. we can also skip the braces and the word return.